docs/installation/install-livepeer/binary-release.md

```diff
@@ -32,6 +32,9 @@ tar -zxvf livepeer-linux-amd64.tar.gz
 
 # Finally, move it to the appropriate directory
 mv livepeer-linux-amd64/* /usr/local/bin/
+
+# Currently, packages are available for Arch Linux
```

Comment:
IMO this should be under a header for Third-Party Packages for the time being - obviously I trust that you're not doing anything nefarious but that might not be the case for everyone

# Third-party packages
Several binaries are maintained by Livepeer community members. Before using these packages, please verify that they have been updated to use the latest builds of `go-livepeer`. We will make best efforts to update this list, but we cannot guarantee that individual packages are up-to-date.

In the future, Livepeer core contributors may produce official binaries for the operating systems below.

## Arch Linux 

**Source**
https://aur.archlinux.org/packages/go-livepeer-bin/

**Installation**
```
paru go-livepeer-bin
```

Reply:
Agreed, making a clear distinction in the docs between official and community maintained packages is in the best interest of Livepeer and new Orchestrators. I will close this and create a new PR as suggested.